packages/core/src/components/forms/file-upload.md

```diff
@@ -9,4 +9,18 @@ Wrap that all in a `label` with class `pt-file-upload`.
     you must implement it separately in JS.
 </div>
 
+@## CSS API
+
 @css pt-file-upload
+
+@## JavaScript API
+
+The `FileUpload` component is available in the __@blueprintjs/core__ package. Make sure to review the [general usage docs for JS components](#blueprint.usage).
```

Comment:
I might hold off on this component until 2.0 as we want to rename to `FileInput` #1453. would be weird to ship and deprecate immediately.

Reply:
Yeah thought about that. I could also just call it `FileInput` now. That okay?

Reply:
it's inconsistent and 2.0 is right around the corner... not sure what to do.

Reply:
Prefer to just ship this as `FileUpload` right now.

Reply:
K